app/person-escort-record/controllers.js

```diff
@@ -0,0 +1,30 @@
+const wizard = require('hmpo-form-wizard')
+
+const FormWizardController = require('../../common/controllers/form-wizard')
+
+function sectionHandler(req, res, next) {
+  const { questions } = req.framework
+  const { key: sectionKey, steps: sectionSteps } = req.frameworkSection
+  const wizardConfig = {
+    buttonText: 'actions::save_and_continue',
+    controller: FormWizardController,
+    entryPoint: true,
+    journeyName: `person-escort-record-${sectionKey}`,
+    journeyPageTitle: 'Person escort record',
```

Comment:
Get from `locales`? (Or am I misremembering what `journeyPageTitle` is used for?)

Reply:
Yeah, good point. Might need a person-escort-record.json file...